### ISSUE TITLE
No Admin Tool and undefined error

### ISSUE THREAD

**Issue by alkaramba** [OfficeDev/microsoft-teams-apps-incentives#6]
Hello,

I have an issue with that app.
I followed the deployment guide.
Here you can see the SharePoint Lists:
![image](https://user-images.githubusercontent.com/65603850/82327067-245bb200-99de-11ea-9745-adb69d954f32.png)

Here you can see the App in PowerApps:
![image](https://user-images.githubusercontent.com/65603850/82327003-01310280-99de-11ea-917b-9c848ee89411.png)

And as Owner of the App and Owner of the SharePoint site, I thought that I can see the Button "Admin Tool" to configure the awards. However, I get this error message instead:
![image](https://user-images.githubusercontent.com/65603850/82327850-4d307700-99df-11ea-8bbd-5d0eff6c1d1c.png)

Does anyone have a solution?

Kind regards
Alex

**Comment by Srivatsan0102** [comment]
Firstly, could you explain if you created the sharepoint lists using the powershell script or by creating them manually? Based on your Power App screenshot, it is showing errors with respect to the Sharepoint lists and you may not have imported them correctly. Double check the data source step, and if possible try and create the lists using powershell script as it should make the deployment a lot easier.

**Comment by alkaramba** [comment]
I added them using the PowerShell Script. I got errors while executing the script because of the attachment column that was already existing.

**Comment by Srivatsan0102** [comment]
In the Powerapp, make sure that you have imported the sharepoint lists from the sharepoint site created from the powershell. Once you do that, you have to add only the group-ID to the formula bar, as opposed to the entire team link.
![AdminGroup9](https://user-images.githubusercontent.com/63128678/83812539-9257d880-a670-11ea-8a6d-8890ce9a9c71.png)

The steps to get the group ID is mentioned in the deployment guide:
https://github.com/OfficeDev/microsoft-teams-apps-incentives/wiki/Deployment-guide

In the case of the admin tool, the visibility of it may be set to false in the powerapp. Under User Screen, select the Home Button1, Admin Tool Button1, Icon27 and Icon26 and set the visibility to on for each.
![issueincentive1](https://user-images.githubusercontent.com/63128678/83812822-04c8b880-a671-11ea-9098-92de88763a01.png)

In case you're unable to edit the visibility component, select the formula option that should appear when hovering over the visibility component and remove the showadmin from the formula bar. You should be able to update the visibility now.

**Comment by Srivatsan0102** [comment]
@Gcotogno From your screenshot, it seems that the home and admin tool are visible, so once the visibility is set to true for the components listed above, save and publish the app again and you should be able to access the tabs from your team.

If you have created the lists manually, follow the steps in the troubleshooting guide to create the lists. https://github.com/OfficeDev/microsoft-teams-apps-incentives/wiki/Troubleshooting#problems-related-to-sharepoint-list-creation

Basically, you will have to select add a new column option -> select more -> then add the column name, type and description. Creating the columns this way will ensure no errors when you import the sharepoint lists to the powerapp studio.
